### ISSUE TITLE
Will a dose decision table be available for model-assissted method like BOIN, mTPI, etc?

### ISSUE THREAD

**Issue by drizztxx** [brockk/escalation#6]
Hi Kristian,

I think a decision table will be very helpful for trial implementation and communication with non-statisticians. Will it be added in the future release?

Thanks,
Liang

**Comment by brockk** [comment]
Can you please give me an example of what you mean? Thanks

**Comment by drizztxx** [comment]
Yeah. Just like [BOIN](https://cran.r-project.org/web/packages/BOIN/index.html), a decision table can be generated as below:
bound <- get.boundary(target=0.3, ncohort=10, cohortsize=3)
summary(bound)
Escalate dose if the observed DLT rate at the current dose <=  0.2364907 
Deescalate dose if the observed DLT rate at the current dose >  0.3585195 

This is equivalent to the following decision boundaries
                                                     
Number of patients treated 3 6 9 12 15 18 21 24 27 30
Escalate if # of DLT <=    0 1 2  2  3  4  4  5  6  7
Deescalate if # of DLT >=  2 3 4  5  6  7  8  9 10 11
Eliminate if # of DLT >=   3 4 5  7  8  9 10 11 12 14
